README.md

```diff
@@ -32,15 +32,16 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
-1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+1. If you were to describe semantic HTML to the next cohort of students, what would you say? Semantic HTML is the use of HTML markup code that helps reinforce the meaning of the information that is in webpages than it just defining the look.
 
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
-
+one big difference is block will create a new line where inline won't. Another difference is that inline doesn't make a block around the display like block does.
 3. What are the 4 areas of the box model?
-
+We got margin, padding, content, and the border.
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
+The align-items works off of the cross axis so the vertical one and so it would center the box vertically.
 
-5. Explain why git is valuable to a team of developers.
+5. Explain why git is valuable to a team of developers. Git helps to clone and pull from github so that we can work on a project locally instead of remotely.
```

Comment:
While this is not an incorrect answer, I just want to stress that git does so much more than that. The main reason it is used is for version contorl. If a mistake is made, or a bug is found, it is extreamly easy to "roll back" to the previous working version. Also github is not the only platform that hosts git repos. Good Job on these questions.